core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2693,6 +2693,9 @@ class ReplicaManagerTest {
     } else {
       assertTrue(stray0.isInstanceOf[HostedPartition.Online])
     }
+
+    replicaManager.shutdown()
+    logManager.shutdown()
```

Comment:
we should probably do it in "finally" otherwise we we may not close them if assertion fails or an exception is thrown

Reply:
You're right! Updated.